src/main.cpp

```diff
@@ -2588,8 +2514,8 @@ bool CBlock::ConnectBlock(CTxDB& txdb, CBlockIndex* pindex, bool fJustCheck, boo
                 vtx[1].GetHash().ToString().substr(0,10));
         }
 
-        //9-3-2015
-        int64_t nMaxResearchAgeReward = NN::ResearchAgeComputer::MaxReward(nTime);
+        // TODO: determine max reward from accrual computer implementation:
+        int64_t nMaxResearchAgeReward = 12750 * COIN;
```

Comment:
Why did you revert this to a hardcoded constant?

Reply:
The `ResearchAgeComputer` type is now internal to the implementation of the tally interface, so `MaxReward()` isn't exposed directly. I didn't want to pull in the more complex changes from this branch just for the fix to compile, so I put in the hard-coded constant until the rest of the superblock window changes are finished.

Reply:
Makes sense.